### PR TITLE
devise configure 

### PR DESCRIPTION
**Pull request by masataka-ohara**
devise設定途中　10/08時点

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
```
Comment (freezer530000) [inline app/controllers/home_controller.rb]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: app/helpers/home_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (freezer530000) [inline app/helpers/home_helper.rb]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150924122432_add_devise_to_users.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+class AddDeviseToUsers < ActiveRecord::Migration
```
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: spec/views/home/show.html.slim_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "home/show.html.slim", type: :view do
```
Comment (freezer530000) [inline spec/views/home/show.html.slim_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/migrate/20150924122432_add_devise_to_users.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  def self.up
```
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:2]: **[reek]**
- [TooManyStatements] has approx 12 statements

File: spec/views/home/index.html.slim_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "home/index.html.slim", type: :view do
```
Comment (freezer530000) [inline spec/views/home/index.html.slim_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: db/migrate/20150924122432_add_devise_to_users.rb [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    change_table(:users) do |t|
```
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: spec/controllers/home_controller_spec.rb [hunk 8]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Extra empty line detected at block body beginning.

File: app/helpers/home_helper.rb (hunk 2 above)
Comment (freezer530000) [inline app/helpers/home_helper.rb]: **[rails_best_practices]**
- remove empty helpers [:ref](http://rails-bestpractices.com/posts/2011/04/09/remove-empty-helpers/)

File: spec/controllers/home_controller_spec.rb [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  describe "GET #index" do
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/controllers/home_controller_spec.rb [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+    it "returns http success" do
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/controllers/home_controller_spec.rb [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #show" do
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/controllers/home_controller_spec.rb [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+    it "returns http success" do
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/controllers/home_controller_spec.rb [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+      get :show
+      expect(response).to have_http_status(:success)
+    end
+  end
+
```
Comment (freezer530000) [inline spec/controllers/home_controller_spec.rb]: **[rubocop]**
- Extra empty line detected at block body end.

**Comment by bakunyo** [comment]
LGTM
LTGM

**Comment by bakunyo** [comment]
## 確認方法
1. このブランチをチェックアウトする
2. `rails console`を立ち上げ、User.createで適当にユーザーつくる
3. サーバーを立ち上げ、/sample にいくと左下にログインのリンク
4. ログインする
5. Rootに飛ばされるけど、 /sample にいくとログインリンクが無くなりログアウトのリンクになってる
6. ログアウトもできることを確認

あとは気になる人は色々いじってみてください〜

**Comment by bakunyo** [comment]
LGTM(2回目)

File: db/migrate/20150924122432_add_devise_to_users.rb (hunk 3 above)
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment [:ref](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md)

File: db/migrate/20150924122432_add_devise_to_users.rb (hunk 5 above)
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:2]: **[reek]**
- [TooManyStatements] has approx 12 statements [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md)

File: db/migrate/20150924122432_add_devise_to_users.rb (hunk 7 above)
Comment (freezer530000) [inline db/migrate/20150924122432_add_devise_to_users.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

**Pull request merged**